staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -88,6 +88,13 @@ const (
 	// validation, merging, mutation can be tested without
 	// committing.
 	DryRun utilfeature.Feature = "DryRun"
+
+	// owner: @kevtaylor
+	// alpha: v1.11
+	//
+	// Allow subpath environment variable substitution
+	// Only applicable if the VolumeSubpath feature is also enabled
+	VolumeSubpathEnvExpansion utilfeature.Feature = "VolumeSubpathEnvExpansion"
```

Comment:
do we need to update the staging file, or is just this one sufficient? https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go

Reply:
I did the staging one because it said so in the api guide but it wasnt totally clear how the 2 are related.

Reply:
I don't think the staging one is necessary.  Can you try removing it?

Reply:
reverted - seems to behave itself regarding feature gates